app/gate/index.js

```diff
@@ -13,17 +13,17 @@ const auth = {
   checkUser: () => api.get('check/user'),
   checkVer: () => api.get('check/ver'),
 
-  signIn: data => api.post('sign-in', data),
-  signInConfirmation: data => api.post('sign-in-confirmation', data),
+  signIn: (data) => api.post('sign-in', data),
+  signInConfirmation: (data) => api.post('sign-in-confirmation', data),
```

Comment:
Why these parentheses are back? I remember it's a rule on ESLint v5 that a single argument for an arrow function doesn't need parentheses.

Reply:
I think this rule is changed! I'm not sure
could you please check here: https://eslint.org/docs/rules/arrow-parens ?

Reply:
seems it's because of prettier 2 changes:
https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-arrowparens-to-always-7430httpsgithubcomprettierprettierpull7430-by-kachkaevhttpsgithubcomkachkaev